lib/screens/create_region_event_screens/interactor/viewmodels/create_region_event_bloc.dart

```diff
@@ -2,13 +2,60 @@ import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
 import 'package:seeds/domain-shared/page_command.dart';
 import 'package:seeds/domain-shared/page_state.dart';
+import 'package:seeds/screens/create_region_event_screens/interactor/viewmodels/create_region_events_page_commands.dart';
 
 part 'create_region_event_events.dart';
 
 part 'create_region_event_state.dart';
 
 class CreateRegionEventBloc extends Bloc<CreateRegionEventEvents, CreateRegionEventState> {
   CreateRegionEventBloc() : super(CreateRegionEventState.initial()) {
+    on<OnNextTapped>(_onNextTapped);
+    on<OnBackPressed>(_onBackPressed);
+    on<OnRegionEventNameChange>(_onRegionEventNameChange);
     on<ClearCreateRegionEventPageCommand>((_, emit) => emit(state.copyWith()));
   }
+
+  void _onRegionEventNameChange(OnRegionEventNameChange event, Emitter<CreateRegionEventState> emit) {
+    emit(state.copyWith(eventName: event.eventName));
+  }
+
+  void _onNextTapped(OnNextTapped event, Emitter<CreateRegionEventState> emit) {
+    switch (state.createRegionEventScreen) {
+      case CreateRegionEventScreen.selectLocation:
+        emit(state.copyWith(createRegionEventScreen: CreateRegionEventScreen.displayName));
+        break;
+      case CreateRegionEventScreen.displayName:
+        emit(state.copyWith(createRegionEventScreen: CreateRegionEventScreen.addDescription));
+        break;
+      case CreateRegionEventScreen.addDescription:
+        emit(state.copyWith(createRegionEventScreen: CreateRegionEventScreen.selectBackgroundImage));
+        break;
+      case CreateRegionEventScreen.selectBackgroundImage:
+        emit(state.copyWith(createRegionEventScreen: CreateRegionEventScreen.reviewAndPublish));
+        break;
```

Comment:
@gguij004 @gguijarro-c-chwy 
Maybe I'm not a Desingner but .. man I thing and feel that set the name, description and Image (no enoght space make that screen scrollable we don need move the user throught several screens) can be done in just one screen then just a navigate to reviewAndPublish

Reply:
Just look at this screen guys it is exactly what I means but add the image below.
![image](https://user-images.githubusercontent.com/42857405/158498874-d7c92d29-4b37-4870-8270-b7c094e9ad6c.png)

Reply:
I think this is a question for @JGDoh . Pls post it in the channel.